package.json

```diff
@@ -1,20 +1,23 @@
 {
   "name": "pretender",
   "version": "2.1.0",
-  "main": "./pretender.js",
+  "main": "./dist/pretender.js",
+  "module": "./src/pretender.es.js",
   "description": "Pretender is a mock server library for XMLHttpRequest and Fetch, that comes with an express/sinatra style syntax for defining routes and their handlers.",
   "license": "MIT",
   "engines": {
     "node": ">=4"
   },
   "scripts": {
+    "prepublishOnly": "npm run build && npm run tests-only",
     "pretest": "bower install",
+    "build": "rollup --config",
     "test": "npm run lint && npm run jscs && npm run tests-only",
-    "test-ci": "npm run pretest && npm run lint && npm run jscs && npm run tests-only-ci",
+    "test-ci": "npm run pretest && npm run build && npm run lint && npm run jscs && npm run tests-only-ci",
```

Comment:
Making sure that the build step is ran before running the tests, in case there is a PR that didn't include the build outputs.